src/components/sortable.jsx

```diff
@@ -255,7 +255,7 @@ var Sortable = React.createClass({
         layout: React.PropTypes.oneOf([HORIZONTAL, VERTICAL]),
         padding: React.PropTypes.bool,
         disabled: React.PropTypes.bool,
-        constraints: React.PropTypes.object,
+        constraints: React.PropTypes.shape({}),
```

Comment:
`.object` is forbidden by the lint rule, so this was a bit of a hacky way around that tbh. I think `any` is probably better.